src/Uno.UWP/Globalization/Language.wasm.cs

```diff
@@ -0,0 +1,19 @@
+﻿#if __WASM__
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.Globalization
+{
+	public partial class Language
+	{
+		public static string CurrentInputMethodLanguageTag { get; private set; } = "";
+
+        public static bool TrySetInputMethodLanguageTag(string languageTag)
```

Comment:
We'll probably need to adjust that to get the browser cultures, but for now, not throwing is best :)